src/main/java/fi/otavanopisto/kuntaapi/server/index/AbstractIndexHander.java

```diff
@@ -40,19 +46,44 @@
   
   @Inject
   private Logger logger;
-
+  
+  @Inject
+  private GenericHttpClient httpClient;
+  
   private String index;
   private TransportClient client;
   
+  
   @PostConstruct
   public void init() {
+    debugPrint("http://localhost:9300");
+    debugPrint("http://localhost:9200");
+    
     String[] hosts = systemSettingController.getSettingValues(KuntaApiConsts.SYSTEM_SETTING_ELASTIC_SEARCH_HOSTS, DEFAULT_HOSTS);
     String clusterName = systemSettingController.getSettingValue(KuntaApiConsts.SYSTEM_SETTING_ELASTIC_CLUSTER_NAME, DEFAULT_CLUSTERNAME);
     index = systemSettingController.getSettingValue(KuntaApiConsts.SYSTEM_SETTING_ELASTIC_INDEX, DEFAULT_INDEX);
     client = createTransportClient(hosts, clusterName);
     setup();
   }
   
+  private void debugPrint(String url) {
+    try {
+      CloseableHttpClient httpClient = HttpClients.createDefault();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename "httpClient" which hides the field declared at line 50. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AHiddenFieldCheck)